README.md

```diff
@@ -24,6 +24,11 @@ $> cp sample.env .env
 $> npm start
 ```
 
+## Documentation
+
+- [Deployment](docs/Deployment.md)
```

Comment:
zomg i would love to have a debate with you and the team about that idea. my feels re: it and docs are much lesser than on other topics but i think this could be a fun 'lunch and learn' topic, cc @cadecairos @simonwex